controllers/clusterversion_controller.go

```diff
@@ -107,8 +109,10 @@ func (r *ClusterVersionReconciler) ensureConsolePlugin(clusterVersion string) er
 	}
 
 	// Create/Update ODF console ConsolePlugin
-	odfConsolePlugin := console.GetConsolePluginCR(r.ConsolePort, basePath, OperatorNamespace)
+	odfConsolePlugin := console.GetConsolePluginCR(r.ConsolePort, OperatorNamespace)
 	_, err = controllerutil.CreateOrUpdate(context.TODO(), r.Client, odfConsolePlugin, func() error {
+		logger.Info(fmt.Sprintf("Set the BasePath for odf-console plugin as '%s'", basePath))
```

Comment:
I don't like this log message here, since this will lead to potentially massive spam in our logs. If you really want to capture this event, first check to see if the current basePath value differs from our expected value, and only then output the log message.